FileConventions/Library.fs

```diff
@@ -0,0 +1,9 @@
+﻿module FileConventions
+
+open System.IO
+open System
+
+let HasShebang (fileInfo: FileInfo) =
+    let fileText = File.ReadAllText(fileInfo.FullName)

```

Comment:
now that you have the simpler code, now let's make an optimization: if the file here being read is very big, then it's better to only read the 1st line instead of loading the whole file in memory; for that, we're going to use a different function from File class

Reply:
Done